src/amp-story-ad-preview.js

```diff
@@ -76,6 +76,20 @@ export default class AmpStoryAdPreview {
     // TODO: Either:
     // a) purifyHtml() from ampproject/src/purifier
     // b) reject when invalid
+    dirty = dirty.replace(
```

Comment:
Maybe extract this replace into a module-level arrow-function? This will allow readers to understand the intent:

```js
const insertHttpsCircumventionPatch = docStr =>
  docStr.replace('<head>', /* ... */);
```